core/themes/basis/css/skin.css

```diff
@@ -1189,7 +1189,7 @@ div.messages.warning:before {
 
 /* Error status message colors */
 .messages.error {
-  color: #f00;
+  color: #9c1d0d;
   background-color: #f9edec;
 }
```

Comment:
I think we can chop out this whole section and let it fall back to the system settings (which are the same), right @herbdool?

Reply:
I'll take your word on it @laryn.